packages/shared/src/lib/core/layer-2/helpers/calculateGasFeeInGlow.ts

```diff
@@ -1,10 +1,10 @@
 import { BigIntLike } from '@ethereumjs/util'
 import { WEI_PER_GLOW } from '../constants/wei.constants'
 
-export function calculateGasFeeInGlow(gasAmount: BigIntLike, gasPriceInWei: BigIntLike | undefined): BigIntLike {
+export function calculateGasFeeInGlow(gasAmount: BigIntLike, gasPriceInWei: BigIntLike | undefined): bigint {
     if (gasAmount && gasPriceInWei) {
         const gasPriceInGlow = BigInt(Number(gasPriceInWei)) / WEI_PER_GLOW
-        return BigInt(Number(gasAmount) * Number(gasPriceInGlow)).toString()
+        return BigInt(Number(gasAmount) * Number(gasPriceInGlow))
```

Comment:
would be better to convert gasAmount to bigint using converter before calculation instead of working in numbers and losing precision?

Reply:
No, because we are multiplying with a factor of 1.1 in `canAccountMakeEvmTransaction`, which is not possible with big ints.